taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -124,3 +124,32 @@ def fill_email_data(config, tasks):
             email["subject"] = email["subject"].format(**format_kwargs)
 
         yield task
+
+
+@transforms.add
+def filter_ytp_tests(config, tasks):
+    """Youtube-playback AV1 and VP9 tests should run on P2 only"""
+    for task in tasks:
+        test_platform = task['treeherder']['platform']
+        test_name = task['name']
+        if "AV1" in test_name or "VP9" in test_name:
```

Comment:
Looking at the tests, these AV1 and VP9 tests are passing on both devices, we can remove this restriction now.

Also, note that this transform didn't work because after a `yield` call, the code returns to the same point when we are generating the next task. You need to call `continue` after the `yield` if you don't want it to continue.

Reply:
I guess i can remove the transform method at all.

Reply:
Yes, let's remove it completely.